Library/Homebrew/tap.rb

```diff
@@ -229,7 +229,7 @@ def install(options = {})
       raise
     end
 
-    link_manpages
```

Comment:
I'm ok with it being one method after all but let's call it `link_additional_files` or similar rather than `repair` anything